internal/konnect/license/client.go

```diff
@@ -48,7 +48,7 @@ func NewClient(cfg adminapi.KonnectConfig) (*Client, error) {
 
 	return &Client{
 		address:        cfg.Address,
-		runtimeGroupID: cfg.RuntimeGroupID,
+		runtimeGroupID: cfg.ControlPlaneID,
```

Comment:
```suggestion
		controlPlaneID: cfg.ControlPlaneID,
```
could we have it renamed here too?

Reply:
I intended to separate the user-facing flag's change from the internal renames. There will be one more PR that will handle all the names/comments, etc. Would that be fine? Or do you prefer all changes at once?